src/main/java/com/minecolonies/coremod/entity/ai/mobs/util/BarbarianSpawnUtils.java

```diff
@@ -42,7 +42,7 @@
     /**
      * Values used in Spawn() method
      */
-    private static final float WHOLE_CIRCLE = 360.0F;
+    private static final double WHOLE_CIRCLE = 360.0;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "WHOLE_CIRCLE" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)